TheWarWithin/Priorities/RogueSubtlety.simc

```diff
@@ -14,13 +14,14 @@ actions.precombat+=/stealth
 actions+=/stealth
 actions+=/kick
 actions+=/variable,name=snd_condition,value=buff.slice_and_dice.up
+actions+=/eviscerate,if=combo_points>=1&!variable.snd_condition
 actions+=/call_action_list,name=cds
 actions+=/call_action_list,name=items
-actions+=/slice_and_dice,if=combo_points>=1&!variable.snd_condition
 actions+=/run_action_list,name=stealthed,strict=1,if=stealthed.all
 # Check if you should use dance, vanish or shadowmeld to enter stealth
 actions+=/call_action_list,name=stealth_cds
-actions+=/call_action_list,name=finish,strict=1,if=buff.darkest_night.up&combo_points=cp_max_spend|effective_combo_points>=cp_max_spend&!buff.darkest_night.up|(combo_points.deficit<=1+talent.deathstalkers_mark|fight_remains<=1&effective_combo_points>=3)&!buff.darkest_night.up
+actions+=/call_action_list,name=finish,if=combo_points>=cp_max_spend&!buff.darkest_night.up
+actions+=/call_action_list,name=finish,if=(combo_points.deficit<=1+talent.deathstalkers_mark|fight_remains<=1&combo_points>=3)&!buff.darkest_night.up
```

Comment:
It is better to merge these conditions (separated by `|`) so that the addon only has to traverse the `finish` list 1x.

`combo_points>=cp_max_spend&!buff.darkest_night.up|((combo_points.deficit<=1+talent.deathstalkers_mark|fight_remains<=1&combo_points>=3)&!buff.darkest_night.up)